src/components/Card/_card.scss

```diff
@@ -12,6 +12,12 @@ $iot-header-padding: $spacing-05;
   display: flex;
   flex-direction: column;
   overflow: hidden;
+
+  &__selected {
+    border: solid $spacing-01 $interactive-02;
+    box-sizing: content-box;
+    margin: -$spacing-01;
```

Comment:
Is this negative margin necessary? I have it currently implemented in DashboardEditor and it bleeds the card into the upper div

![Screen Shot 2020-11-04 at 3 21 52 PM](https://user-images.githubusercontent.com/25177649/98169224-7ec7cc80-1eb1-11eb-9be4-5216f965c915.png)

example can be seen here: https://deploy-preview-1751--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-experimental-dashboardeditor--default

Reply:
you get a jump in the card layout otherwise. Can look into it

Reply:
Removing negative margins for this and the resize.